Cargo.toml

```diff
@@ -1374,9 +1374,9 @@ substrate-test-runtime-client = { path = "substrate/test-utils/runtime/client" }
 substrate-test-runtime-transaction-pool = { path = "substrate/test-utils/runtime/transaction-pool" }
 substrate-test-utils = { path = "substrate/test-utils" }
 substrate-wasm-builder = { path = "substrate/utils/wasm-builder", default-features = false }
-subxt = { version = "0.38.1", default-features = false }
-subxt-metadata = { version = "0.38.0", default-features = false }
-subxt-signer = { version = "0.38" }
+subxt = { version = "0.42.0", default-features = false }
+subxt-metadata = { version = "0.42.0", default-features = false }
+subxt-signer = { version = "0.42.0" }
```

Comment:
Bumped Subxt to the latest version, which supports V16 metadata via frame-metadata 23.0.0 and so hopefully will avoid an error I saw in CI re unable to decode metadata.